src/stories/EtherscanLink.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
```

Comment:
Regarding the placement of this story.
Do you intend to follow the pattern of having all stories inside `src/stories/` or would you rather have it inside the component's folder?

I prefer to later, to keep it closer to the component definition.

Reply:
For now, as we have a looot of components and not so many stories, I would prefer stories together in one folder.
But that can change.
@anxolin, @W3stside opinions?